shell/lisa_shell

```diff
@@ -143,6 +143,20 @@ function _lisa-upgrade-pip {
     fi
 }
 
+function _lisa-install-nbextensions {
+    if which nodejs &>/dev/null; then
+        echo
+        echo "Enabling ipywidget extensions..."
+        jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

Comment:
Then we should probably do the same for the list of `.deb` dependencies... cannot we keep this change for when it will be needed? TBH I don't know if there are or will be additional extensions worth to have...

Reply:
> Then we should probably do the same for the list of .deb dependencies

These are not touched by lisa-install, but yes, we could show a warning. That said, we really don't add often distro packages.

> cannot we keep this change for when it will be needed?

Actually yes, since the commit sha1 when lisa-install was run last is already recorded. We just have to remember to do that, or people will complain that LISA is broken.

Reply:
This feature will be added in the future... ;)